### ISSUE TITLE
VariableSizeGrid resize issue when used in combination with AutoSizer

### ISSUE THREAD

**Issue by metacurb** [bvaughn/react-window#792]
**tl;dr: Resizing when using VariableSizeGrid in combination with AutoSizer doesn't update the width of the inner container. Demo here - https://codesandbox.io/p/sandbox/relaxed-tree-hjjp98**

--- 

I am running into an issue with VariableSizeGrid and AutoSizer. I am potentially using it incorrectly, but figured I'd ask here because I can't seem to work out a solution.

My scenario is that I am trying to:
* Create a full height and width container 
* Allow the number of columns to be dynamically controlled by the parent
* Ensure that the number of items per column always fill the container
* Have no horizontal scroll; only vertical

When the page first loads, everything is perfect. The cards fit horizontally based on the columns supplied, and there is no horizontal scroll. However, if I resize the window, the width of the inner container isn't updating.

| Initial | After resize |
|--------|--------|
| <img width="646" alt="image" src="https://github.com/user-attachments/assets/c2322a93-a30d-4344-a415-8775cc6d255e"> | <img width="821" alt="image" src="https://github.com/user-attachments/assets/a99e7a73-0811-4ea1-8c84-c9ba9495f545"> | 

You can see that the container (which I've set to grey) is taking the full width and height.
Taking a look further, I can see that the inner div (setting the height and width of all items) is correct initially. On resize, the width is not being updated. 

Parent is 625px going up 834px, but the inner container stays at 625px.

| Initial | After resize |
|--------|--------|
| <img width="495" alt="image" src="https://github.com/user-attachments/assets/d38f3d78-cc47-403b-89a2-2de77f66bf32"> | <img width="508" alt="image" src="https://github.com/user-attachments/assets/fcd76a71-3874-4ada-bcbd-d3bf96268efb"> | 

Any help to identify where I'm going wrong would be much appreciated!

**Comment by functoria** [comment]
Hi, I don't think this will work because you need to reset the measurements cache after item resize: see `resetAfterColumnIndex` [here](https://react-window.vercel.app/#/api/VariableSizeGrid). 

Since in your case the items seem to be identical in size, why not using `FixedSizeGrid` instead?
